docs/copy-secrets-to-vault.md

```diff
@@ -0,0 +1,58 @@
+# `copy-secrets-to-vault`
+This `vault-radar` command is used for copying unVaulted secrets discovered in a supported scan into HashiCorp Vault.
+
+## Usage
+
+### Supported Scans
+
+We can copy secrets from scans that support this functionality. Those are currently:
+
+- `scan-folder`
+- `scan-aws-s3`
+
+If you'd like additional scan types to be added, please reach out and give us that feedback!
+
+### A quick intro/setting the scene
+
+`copy-secrets-to-vault` copies unVaulted secrets, discovered in a scan, to HashiCorp Vault. As a result, it needs to know which secrets are already vaulted.
+
+It figures this out by looking at the outfile of a previous supported scan that used a Vault index file. For example, if you ran
+
+```bash
+vault-radar scan-folder -o scan-folder-outfile --index-file vault-scan-outfile
+```
+
+where `vault-scan-outfile` is the result of a `scan-vault`, this command would accept as input `scan-folder-outfile` to understand which secrets need to be vaulted.
```

Comment:
It seems like there are potentially a few steps here. E.g. create an index file from vault, run a scan with that index file in order to get the result `scan-folder-outfile` that's used as input. 

I'm wondering if it'd be nice to outline these as set up steps w/ the full commands. But I may be totally misunderstanding this.

Reply:
good suggestion. how's this? aae953d